file.c

```diff
@@ -5363,61 +5363,183 @@ test_check(int n, int argc, VALUE *argv)
 #define CHECK(n) test_check((n), argc, argv)
 
 /*
+ *  :markup: markdown
```

Comment:
```suggestion
```

Reply:
On my machine, removing this line breaks the table (not rendered as an HMTL table).  I don't have make, and so I generate doc with `rdoc --op html`.  Is it possible that using make gives a different result?